tests/BenchmarkDotNet.IntegrationTests/JoinedBenchmarksTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Order;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Tests.Loggers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace BenchmarkDotNet.IntegrationTests
```

Comment:
big thanks for adding the test! It looks like the benchmarks should never be executed in this case so it can be moved to the unit tests project (we try to keep the number of intergration tests low)